### ISSUE TITLE
Functional Dropout in evaluation mode

### ISSUE THREAD

**Issue by felentovic** [Coderx7/SimpleNet#2]
Hello,
a trained network in the evaluation mode will give a different output on every run for the same example because of the functional dropout, `torch.nn.F.Dropout ` in the method `forward()` with the variable `training` set to `True`.

```
def forward(self, x):
        #print(x.size())
        out = self.features(x)

        #Global Max Pooling
        out = F.max_pool2d(out, kernel_size=out.size()[2:]) 
        out = F.dropout2d(out, 0.1, training=True)

        out = out.view(out.size(0), -1)
        out = self.classifier(out)
        return out
```
The solution will be to use `torch.nn.Dropout` instead and I think it should be defined in `__init__()` method.
This answer explains `torch.nn.Dropout` vs `torch.nn.F.Dropout ` : [https://stackoverflow.com/questions/53419474/nn-dropout-vs-f-dropout-pytorch](url)

**Comment by Coderx7** [comment]
Thanks a lot for pointing this out. I completely missed that. it should be fixed now.
